spec/features/request_options_spec.rb

```diff
@@ -3,71 +3,46 @@
 require 'rails_helper'
 
 describe 'Request Options' do
-  before { stub_holding_locations }
+  before { stub_alma_holding_locations }
 
   describe 'the request page', js: true do
     before do
-      visit '/catalog/9618072'
-    end
-
-    it 'clicking the request button loads the request page' do
-      stub_request(:get, "https://bibdata-staging.princeton.edu/bibliographic/9618072/holdings/9455965/availability.json")
+      stub_request(:get, "#{Requests.config['bibdata_base']}/bibliographic/99113436223506421/holdings/22553672700006421/availability.json")
         .to_return(status: 200, body: {}.to_json, headers: {})
-      stub_request(:get, "https://catalog.princeton.edu/catalog/9618072/raw")
+      stub_request(:get, "https://catalog.princeton.edu/catalog/99113436223506421/raw")
         .to_return(status: 200, body: {}.to_json, headers: {})
-      stub_request(:get, "https://bibdata-staging.princeton.edu/availability?id=9618072")
+      stub_request(:get, "#{Requests.config['bibdata_base']}/availability?id=99113436223506421")
         .to_return(status: 200, body: {}.to_json, headers: {})
-      stub_request(:get, "https://bibdata-staging.princeton.edu/locations/delivery_locations.json")
+      stub_request(:get, "#{Requests.config['bibdata_base']}/locations/delivery_locations.json")
         .to_return(status: 200, body: {}.to_json, headers: {})
-      using_wait_time 5 do
-        click_link('Request')
-        expect(current_path).to eq "/requests/9618072"
-      end
+      visit '/catalog/99113436223506421'
```

Comment:
Question: Why is this not 9996180723506421?

Reply:
The test is testing that Request button that exists. It doesnt make a difference. I changed to the other id when I couldn't make capybara run successfully thinking that maybe it had to do with a delay on the record.